cpp/src/arrow/compute/api_scalar.h

```diff
@@ -279,6 +279,45 @@ Result<Datum> Power(const Datum& left, const Datum& right,
                     ArithmeticOptions options = ArithmeticOptions(),
                     ExecContext* ctx = NULLPTR);
 
+/// \brief Left shift (logical) the left array by the right array. Array values must
```

Comment:
Since logical and arithmetic left shift are the same, is it useful to precise which one this is?